### DISCUSSION TITLE
Error 500 when i try to change my Profile Picture or the Logo from the Outline App

### DISCUSSION THREAD

**Discussion by Trackhe** [outline/outline#4768] [Self Hosting] [answered]
I deployed outline with docker and installed it on slack and set the 5 permissions as the documentation says.
But i get only the error 500 when i try to change the profile picture or the App Logo in Outline.

In the log from the docker-image fires every time the same error:

Error-Log:
```
Error [CredentialsError]: Could not load credentials from any providers
    at Timeout.connectTimeout (/opt/outline/node_modules/aws-sdk/lib/http/node.js:69:15)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7) {
  code: 'CredentialsError',
  time: 2023-01-24T22:05:20.916Z,
  retryable: true,
  originalError: {
    message: 'EC2 Metadata roleName request returned error',
    code: 'TimeoutError',
    time: 2023-01-24T22:05:20.916Z,
    retryable: true,
    originalError: {
      message: 'Socket timed out without establishing a connection',
      code: 'TimeoutError',
      time: 2023-01-24T22:05:20.916Z,
      retryable: true
    }
  },
  status: 500
}
```

- Install: self hosted
- Version: v0.67.2
- Auth-Provider: Slack

**Comment by tommoor** [comment, marked as answer]
`Socket timed out without establishing a connection` – it looks like Outline could not reach your storage bucket

**Reply by Trackhe** [reply]
i will check that. thank you

**Reply by CHUYKOPS** [reply]
Hey, it's happens if your minio bucket settings set to private. It's ok but not for profile pictures and public icons.
You need to set "Public" and "Avatars" like this in your minio.
![image](https://user-images.githubusercontent.com/88129934/214531012-d5a70ea4-91d7-489a-ac69-44e9143c383d.png)

**Reply by Trackhe** [reply]
I use Outline without the storage bucket(also no minio) xD maybe this is the problem.
i thought this is only needed if i use it with the extra webserver that would deployed by docker-compose. https://wiki.generaloutline.com/s/hosting/doc/docker-7pfeLP5a8t ^^

**Reply by Limezy** [reply]
Without a storage bucket you won't be able to set a logo or profile pic !

**Reply by Trackhe** [reply]
So i deployed minio with docker. 
And after some time and trying something i get it running ^^, i don't know why your screenshot looks not the same as i have but it works now ^^.

Thank you.
